bin/cfndsl

```diff
@@ -51,6 +52,32 @@ optparse = OptionParser.new do |opts|
     options[:disable_binding] = true
   end
 
+  opts.on('-g', '--generate RESOURCE_TYPE,RESOURCE_LOGICAL_NAME', 'Add resource type and logical name') do |r|
+    options[:lego] = true
+    options[:resources] << r
+  end
+
+  opts.on('-i', "--indent TOKEN", "Use TOKEN for indent character (default space)") do |i|
+    options[:lego] = true
+    options[:indent_token] = i
+  end
+
+  opts.on("-l", "--list", "List supported resources") do
+    puts Cfnlego.Resources.sort
+    exit
+  end
+
+  opts.on("-c", "--indent-count [COUNT]", Integer, "Count of characters to use for indenting. (default: 2)") do |count|
+    options[:lego] = true
+    options[:indent_count] = count
+  end
+
```

Comment:
It it possible to combine these two ideas? Allowing the user to pass a string as an indent token would encapsulate the idea of indent as well. The default could be `'  '` (two spaces). Any user that would like to change that can simply pass a different token, rather than change the indent number (which only really makes sense when using spaces anyway).